shared/cfg/base.cfg

```diff
@@ -113,6 +113,15 @@ q35:
     usb_controller = xhci
     usb_type = qemu-xhci
     usb_type_usb1 = qemu-xhci
+#use qemu-xhci as the default controller on aarch64 platform.
+arm64-pci:
```

Comment:
What about just modify L#112 to `q35, arm64-pci, arm64-mmio:`?

Reply:
@vivianQizhu Changed.